### PR TITLE
Misc pipeline fixes

### PR DESCRIPTION
**Pull request by graham-skylight**
* Basically just covering for missing data we didn't anticipate

### CODE REVIEW

File: src/lib/phdi-transforms/phdi_transforms/geo.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def geocode(client: us_street.Client, address: str) -> GeocodeResult:
     lookup = Lookup(street=address)
     client.send_lookup(lookup)
 
-    if lookup.result:
+    if lookup.result and lookup.result[0].metadata.latitude:
```
Comment (DanPaseltiner) [inline src/lib/phdi-transforms/phdi_transforms/geo.py:33]: What's the problem we are trying to solve here? Were we getting back empty results?

**Review by DanPaseltiner — APPROVED** [review]
I don't quite follow the change in geo.py, but trust that with the context of the problem it is solving things would make sense. Beyond that this looks good to me. Perhaps post-prototype we should investigate these occurrences of missing data to make sure it isn't some how a result of the FHIR conversion process.

File: src/lib/phdi-transforms/phdi_transforms/geo.py (hunk 1 above)
Comment (graham-skylight) [reply src/lib/phdi-transforms/phdi_transforms/geo.py:33]: I did a pretty long local run with the other changes, and then got a response where a seemingly valid response had blank lat/lng and crashed out.

**Review by JSkeets — APPROVED** [review]
LGTM

File: src/lib/phdi-transforms/phdi_transforms/geo.py (hunk 1 above)
Comment (DanPaseltiner) [reply src/lib/phdi-transforms/phdi_transforms/geo.py:33]: I figured it was something like that.

**Pull request merged**